authorization/invitation/service/invitation_service_blackbox_test.go

```diff
@@ -34,7 +35,8 @@ func (s *invitationServiceBlackBoxTest) SetupTest() {
 	s.invitationRepo = invitationrepo.NewInvitationRepository(s.DB)
 	s.identityRepo = account.NewIdentityRepository(s.DB)
 	s.orgService = s.Application.OrganizationService()
-	s.Application = gormapplication.NewGormDB(s.DB, s.Configuration, factory.WithWITService(&test.DevWITService{}))
+	s.devNotificationService = &test.DevNotificationService{}
+	s.Application = gormapplication.NewGormDB(s.DB, s.Configuration, factory.WithWITService(&test.DevWITService{}), factory.WithNotificationService(s.devNotificationService))
 }
 
 func (s *invitationServiceBlackBoxTest) TestIssueInvitationByIdentityID() {
```

Comment:
if this test method contains multiple scenarii, could you split them using sub-tests?